frontend/taipy-gui/src/context/taipyReducers.ts

```diff
@@ -379,14 +384,16 @@ export const taipyReducer = (state: TaipyState, baseAction: TaipyBaseAction): Ta
                         message: alertAction.message,
                         system: alertAction.system,
                         duration: alertAction.duration,
+                        notification_id: alertAction.notification_id,
```

Comment:
notification_id: alertAction.notification_id or nanoid(),
for internal use (ie fileSelector)